grype/presenter/format.go

```diff
@@ -9,10 +9,9 @@ const (
 	jsonFormat      format = "json"
 	tableFormat     format = "table"
 	cycloneDXFormat format = "cyclonedx"
```

Comment:
I don't think we should make a breaking change on the user surface area yet. That is, `cyclonedx` should still produce the XML format and probably add a `cyclonedx-json` as well as `cyclonedx-xml` (as done here). Then we can add deprecation warnings around `cyclonedx` usage mentioning that the behavior will change at grype 1.0 and make the swap then. (which means we should add an issue and capture under a grype 1.0 milestone)

Reply:
+1 -- I like being v1.0 at the moment because we can make choices like this while doing a graceful deprecation of things that didn't work. I'll follow the above and we'll put it on the `TODO` pile when we go to release v1.0